src/drawers/arcDrawer.ts

```diff
@@ -39,6 +39,9 @@ export module _Drawer {
     public draw(data: any[], drawSteps: DrawStep[], userMetadata: any, plotMetadata: Plot.PlotMetadata) {
       // HACKHACK Applying metadata should be done in base class
       var valueAccessor = (d: any, i: number) => drawSteps[0].attrToProjector["value"](d, i, userMetadata, plotMetadata);
+
+      data = data.filter(e => !!+valueAccessor(e, null));
```

Comment:
The slice is drawn, but it has size 0. A tooltip would still stick to it for example, but it wouldn't if the data wasn't there at all.